dropwizard-jersey/src/test/java/io/dropwizard/jersey/validation/ConstraintViolationExceptionMapperTest.java

```diff
@@ -548,4 +549,60 @@ public void testInvalidFieldQueryParam() {
         assertThat(response.readEntity(String.class))
             .containsOnlyOnce("sortParam must match \\\"^(asc|desc)$\\\"");
     }
+
+    @Test
+    public void testParameterValidation() {
+        Response response = target("/valid/paramValidation")
+            .request()
+            .get();
+        assertThat(response.getStatus()).isEqualTo(400);
+        assertThat(response.readEntity(String.class))
+            .containsOnlyOnce("query param length may not be null");
+        response = target("/valid/paramValidation")
+            .queryParam("length", 50)
+            .request()
+            .get();
+        assertThat(response.getStatus()).isEqualTo(400);
```

Comment:
Do you mind splitting up each of these assertions into a separate test case with a descriptive name like the other tests found in the file?